### PR TITLE
Add frozen_string_literal pragma to ruby files

### PR DESCRIPTION
**Pull request by krzysiek1507**
According to this blog post: https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/

### CODE REVIEW

File: lib/acts_as_list/active_record/acts/position_column_method_definer.rb [hunk 1]
```diff
@@ -21,11 +23,11 @@ def self.define_class_methods(caller_class, position_column)
       end
 
       define_singleton_method :decrement_all do
-        update_all_with_touch "#{quoted_position_column} = (#{quoted_position_column_with_table_name} - 1)"
+        update_all_with_touch "#{quoted_position_column} = (#{quoted_position_column_with_table_name} - 1)".dup
```
Comment (brendon) [inline lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: Sorry, I'm feeling dense today :) Can you explain why we need to dup the string? :)

**Comment by brendon** [comment]
Thanks @krzysiek1507 :D

File: lib/acts_as_list/active_record/acts/position_column_method_definer.rb (hunk 1 above)
Comment (krzysiek1507) [reply lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: [Here](https://github.com/swanandp/acts_as_list/blob/master/lib/acts_as_list/active_record/acts/position_column_method_definer.rb#L32) this string is concatenated but one cannot change a frozen string. :)

Comment (brendon) [reply lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: Could we instead unfreeze (`.dup`?) `updates` in:

```
update_all(updates.dup << touch_record_sql)
```

?

Comment (krzysiek1507) [reply lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: In this case, we won't mutate `updates` so maybe we can change it to concatenation?
```ruby
update_all(updates + touch_record_sql)
```

Comment (brendon) [reply lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: Yep, that't cleaner :)

Comment (krzysiek1507) [reply lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: @brendon changed! :)

Comment (brendon) [reply lib/acts_as_list/active_record/acts/position_column_method_definer.rb]: Thanks @krzysiek1507, I'll just wait on @swanandp's check and then we'll be GTG.

**Comment by krzysiek1507** [comment]
TravisCI fails on Rails 5.2 and MySQL but it also fails on master.

**Comment by swanandp** [comment]
I just checked, Rails 5.2 is a different issue not related to this one.

**Comment by brendon** [comment]
We seem to be having suite failures for Rails 5.2 and certain rubies. I think it might be to do with rake versions? or something like that. I don't have time to look into it this weekend, but if either of you two can that would be cool :)

**Comment by krzysiek1507** [comment]
The problem is mysql2. It is old and doesn't work with Rails 5.2 but `0.4.10` doesn't work with Rails < 4.2.

**Comment by brendon** [comment]
Ah yes, sorry I already had this fixed in a branch. I'm just running it through the suite with the latest from master then I'll merge it in. Once that's done, can you update from master and we'll see if the suite passes still :)

**Comment by krzysiek1507** [comment]
Hi @brendon! I'll do it! :)

**Comment by brendon** [comment]
@krzysiek1507 Can you update from master, then it should be all green :)

**Comment by krzysiek1507** [comment]
@brendon all green!

**Comment by brendon** [comment]
Thanks @krzysiek1507 :) Great job!

**Comment by krzysiek1507** [comment]
Thank you!

**Pull request merged**